install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml

```diff
@@ -134,6 +134,7 @@ data:
       scheme: https
       tls_config:
         ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        server_name: kubernetes
```

Comment:
we are moving away from the old helm charts, and planning to use https://github.com/istio/istio/tree/master/manifests with `istioctl manifest apply`.

do you know if this issue also exists in https://github.com/istio/istio/tree/master/manifests?

Reply:
yep, even I installed using istioctl manifest apply, I thought istioctl internally talks to helm charts to install add-ons such as Prometheus, hence I made the PR, yeah if that is not the case this exist in manifest too. :) I will make another PR